### ISSUE TITLE
Fixture directory structure

### ISSUE THREAD

**Issue by Cryde** [theofidry/AliceBundle#14]
Hello,

I've struggled with this bundle because i've created a folder structure under the "fixtures" directory.  
```
fixtures/user/user.yaml
fixtures/user/user_type.yaml
// etc
```
But after digging it seems that I can not do that because of : 
```
$files = SymfonyFinder::create()->files()->in($fullPaths)->depth(0)->name('/.*\.(ya?ml|php)$/i');
```
In `vendor/hautelook/alice-bundle/src/Locator/EnvDirectoryLocator.php:85`  

Is there a special reason to limit the depth ?

I will put my files at the root dir

**Comment by theofidry** [comment]
>Is there a special reason to limit the depth ?

Yes! The reason is that it represents environments: https://github.com/theofidry/AliceBundle/blob/master/doc/advanced-usage.md#environment-specific-fixtures

I am not sure how this could be made either better/less confusing :/

**Comment by Cryde** [comment]
Maybe load the environnement-less file first, with depth 0 and the per env with "infinite" depth ?  
Anyway I solve this by simply prefixing files

(maybe we should put in the doc that it won't go in subdir in per env dir ?)

Thanks for your answers

**Comment by isxoqovbegzod** [comment]
kkjh

**Comment by isxoqovbegzod** [comment]
not

**Comment by theofidry** [comment]
To be honest I'm not sure if it's because it was desired or to keep BC. I'm happy to change this but this requires some checking and probably a new major version. Maybe a flag could be added to enable this behaviour and add a deprecation message that this will become the default behaviour

**Comment by d0niek** [comment]
Support of subdirectories will be nice feature! ;-)

Load subdirectories by default (or with flag), env files put in **fixtures/env/{prod,test,etc.}/** directory?
